apps/backend/src/users/update-user.dto.ts

```diff
@@ -0,0 +1,50 @@
+import { Status, Role, Team } from './types';
+import {
+  IsEmail,
+  IsOptional,
+  IsEnum,
+  IsArray,
+  ArrayMinSize,
+  ArrayUnique,
+  IsUrl,
+} from 'class-validator';
+
+export class UpdateUserDTO {
+  @IsOptional()
+  @IsEnum(Status)
+  status?: Status;
+
+  @IsOptional()
+  @IsEmail()
+  email?: string;
+
+  @IsOptional()
+  profilePicture?: string;
+
+  @IsOptional()
+  @IsUrl({
+    protocols: ['http', 'https'],
```

Comment:
⛏️ Very cool! If we do this, I think we should restrict the protocol to just https. Linkedin and github both support https and there's no reason to prefer HTTP over https